tests/test_interact.py

```diff
@@ -62,10 +63,12 @@ def test_interact_escape_None(self):
         p.sendcontrol(']')
         p.expect('29<STOP>')
         p.send('\x00')
-        if not os.environ.get('TRAVIS', None):
-            # on Travis-CI, we sometimes miss trailing stdout from the
-            # chain of child processes, not entirely sure why. So this
-            # is skipped on such systems.
+
+        if not platform.system() == 'Linux':
```

Comment:
I'm not quite happy that this test piece which was previously skipped only on Travis is now skipped on all Linux systems. We presumably had it passing on Linux at some point in the past. Is that no longer the case even without the changes in this PR?